dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <UploadFormSubcategoryHeader label="Activity Name" description="Name of the activity." />
+  <div class="form-field next-to-each-other">
+    <p>
+      <b> {{ selectedActivities ? selectedActivities.name : "" }}</b>
+    </p>
+
+    <FormKit
+      type="hidden"
+      name="activityName"
+      :modelValue="selectedActivities ? selectedActivities.value : ''"
+      disabled="true"
+    />
+
+    <PrimeButton
+      data-test="dataTestChooseActivityButton"
+      :label="selectedActivities ? 'Change Activity' : 'Choose Activity'"
+      class="p-button-text p-0 m-0"
+      :icon="selectedActivities ? 'pi pi-pencil' : 'pi pi-list'"
+      @focus="inputFocused"
+    />
+  </div>
+
+  <OverlayPanel ref="overlayPanel" data-test="activityOverlayPanel">
+    <div>
+      <Tree :value="allActivities" class="w-full md:w-30rem" placeholder="Select Activitie">
+        <template #default="slotProps">
+          <b>{{ slotProps.node.name }}</b>
+        </template>
+        <template #child="slotProps">
+          <span class="next-to-each-other -ml-5">
+            <RadioButton
+              v-model="selectedActivities"
+              :inputId="slotProps.node.reference"
+              name="selectedActivities"
+              :value="slotProps.node"
+              @change="newActivitieSelected"
+            />
+            <label :for="slotProps.node.key" class="ml-2">{{ slotProps.node.name }}</label>
+          </span>
+        </template>
+      </Tree>
+    </div>
+  </OverlayPanel>
+
+  <div class="my-4">
+    <MultiSelectFormField
+      dataTest="selectNaceCodes"
+      name="naceCodes"
+      validation="required"
+      validation-label="Nace Codes for Activity"
+      description="The NACE codes associated with this activity"
+      label="Nace Codes"
+      placeholder="Chose Nace Codes for Activity"
+      :options="NaceCodesForActivities"
```

Comment:
The first letter of a variable name is typically lower case.

Reply:
It is actually a function call but the argument still holds. It should be camel and not pascal case.

Reply:
changed